src/Lightning.ts

```diff
@@ -581,7 +577,7 @@ export const LightningMixin = (superclass) => class extends superclass {
     // we're doing the the Transaction.find after the lock to make sure there is no race condition
     // note: there might be another design that doesn't requiere a lock at the uid level but only at the hash level,
     // but will need to dig more into the cursor aspect of mongodb to see if there is a concurrency-safe way to do it.
-    await redlock({ path: this.user._id, logger: lightningLogger, lock }, async (lock) => {
+    await redlock({ path: this.user._id, logger: lightningLogger, lock }, async () => {
```

Comment:
lock was not used I guess?

Reply:
No. and it was not used in so many other places.